conans/client/cmd/search.py

```diff
@@ -33,7 +33,7 @@ def search_recipes(self, pattern, remote=None, case_sensitive=False):
         references[remote.name] = refs
         return references
 
-    def search_packages(self, reference=None, remote_name=None, query=None, outdated=False):
+    def search_packages(self, reference=None, remote=None, query=None, outdated=False):
```

Comment:
Please use the ``remote_name``, it helps to differentiate to a ``Remote`` object, that it is also used as argument in other places.